pkg/fileutils/read_test.go

```diff
@@ -254,10 +248,57 @@ var _ = Describe("EmptyScheme", func() {
 		})
 	})
 
-	Context("when no scheme and using windows separators and drive letter", func() {
+	Context("when http: is the scheme", func() {
 		BeforeEach(func() {
-			file = "C:\\my\\file"
-			os = "windows"
+			file = "http://foo.com/bar"
+		})
+
+		It("scheme should not be empty", func() {
+			Expect(err).NotTo(HaveOccurred())
+			Expect(empty).To(BeFalse())
+		})
+	})
+
+	Context("when https: is the scheme", func() {
+		BeforeEach(func() {
+			u, err = url.Parse("https://foo.com/bar")
+		})
+
+		It("scheme should not be empty", func() {
+			Expect(err).NotTo(HaveOccurred())
+			Expect(empty).To(BeFalse())
+		})
+	})
+})
+
+var _ = Describe("EmptyScheme on Unix-style OS", func() {
```

Comment:
This should be part of a single "EmptyScheme" Describe block with two Context blocks for macOS and linux, respectively. That will stop the two Describe blocks "parting company".